src/nupic/algorithms/KNNClassifier.py

```diff
@@ -1009,6 +1049,13 @@ def computeSVD(self, numSVDSamples=None, finalize=True):
 
 
   def getAdaptiveSVDDims(self, singularValues, fractionOfMax=0.001):
+    """
+    TODO: Somebody help me write this.
+
+    :param singularValues:
+    :param fractionOfMax:
+    :return:
+    """
```

Comment:
@numenta/nupic-reviewers I need help writing this. You can simply respond in a comment with a function description and I'll do the rest.